services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.tsx

```diff
@@ -331,6 +330,12 @@ export const HealthPlanPackageTable = ({
                                 </tr>
                             ))}
                         </tbody>
+                        <caption className={caption ?? styles.srOnly}>
+                            {caption || 'Table'}
```

Comment:
🔴  Captions have intended usage - they should be [specific to the table content](https://www.w3.org/WAI/tutorials/tables/caption-summary/). E.g. in this case "submissions" would be a good fallback caption for this component.

Also it seems like with your change we are going to always include a caption. So then does that man `caption` prop has changed its meaning? Now instead of showing or caption or not its becoming more `showCsutomCaption` and is used for optionally displaying a custom caption to the user. Is that the intent ?

Reply:
We’re in the roadmap meeting @haworku :p

Reply:
This is a great point out! 

`"Captions should be specific to the table content"`, so maybe we should update it to fallback to `Submission table`. When using VO, without the `table` part it will announce `Submission, 0 filters applied, displaying 2 of 2 submissions`, which sounded confusing and did describe the context of what the submission was in. Hence the `Table` as a default, but zooming out from just the table, the component itself is scoped to a table of submissions so `Submission table` would make more sense.

> Now instead of showing or caption or not its becoming more `showCsutomCaption` and is used for optionally displaying a custom caption to the user. Is that the intent ?

If I'm interpreting your question correctly, yes. I was trying to preserve the use of caption for sighted users too incase we did want to have a custom caption that should be displayed for sighted users. The fallback is mainly used for VO users, since we do not have a caption for the table by default it has to announce something before the filter/submission count otherwise it would sound confusing. Without a `caption` passed, the fallback will still be hidden for sighted users and when one is passed it will be shown. The filter/submission count will always be hidden. As for the prop name `caption` --> `showCustomCaption` I think that it still showing a caption or not for sighted users, but for VO users it will be `showCustomCaption`.

Reply:
Ah, okay so the reason "Submissions" may be enough and we don't need to append "table" is because the screenreader announces the caption and then the word "table" already. This is how captions and things like aria-labels work - they provide an [accessible name](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/#naming_effectively) which is announced, and then then the semantic html name is announced.
> Do NOT include a WAI-ARIA role name in the accessible name. For example, do not include the word "button" in the name of a button, the word "image" in the name of an image, or the word "navigation" in the name of a navigation region. Doing so would create duplicate screen reader output since screen readers convey the role of an element in addition to its name. 

That guidance applies here. `table` is already the role implicitly. We can also do whatever for now and check in our user test about this. 

I'm fine with following whatever you decide about prop naming but that makes sense. Thanks for clarifying the intent just wanted to be sure I wasn't assuming things.